bulk_insert_test.go

```diff
@@ -26,6 +26,78 @@ type fakeTable struct {
 	UpdatedAt time.Time
 }
 
+func TestBulkInsertWithReturningValues(t *testing.T) {
```

Comment:
Still think this test case should be simplified without column options and fields not needed.

Reply:
You mean, we don't need `insert_option` related to https://github.com/t-tiger/gorm-bulk-insert/pull/53#discussion_r745157644 ? Sorry, I could not understand very well.

Reply:
It's a super nit but I just mean the test is copied when testing custom field tags so this test is also using the field tag `gorm:"column:ThisIsCamelCase"`. I think that's confusing since we already test that feature in a different test, this test could use a simpler struct.

```go
type Table struct {
	ID        uint `gorm:"primary_key;auto_increment"`
	ColumnOne string
	ColumnTwo string
}
```

And then just refer to the default column names `column_one` and `column_two`.